app-modules/report/src/Filament/Widgets/UsersLoginCountTable.php

```diff
@@ -0,0 +1,158 @@
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\Report\Filament\Widgets;
+
+use App\Models\User;
+use Filament\Forms\Components\DatePicker;
+use Filament\Support\Enums\MaxWidth;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Filters\Filter;
+use Filament\Tables\Filters\SelectFilter;
+use Filament\Tables\Table;
+use Filament\Widgets\TableWidget as BaseWidget;
+use Illuminate\Database\Eloquent\Builder;
+use Livewire\Attributes\On;
+
+class UsersLoginCountTable extends BaseWidget
+{
+    public string $cacheTag;
+
+    protected static ?string $heading = 'Users Login Count';
+
+    protected static bool $isLazy = false;
+
+    protected static ?string $pollingInterval = null;
+
+    protected int | string | array $columnSpan = [
+        'sm' => 1,
+        'md' => 4,
+        'lg' => 4,
+    ];
+
+    public function mount(string $cacheTag)
+    {
+        $this->cacheTag = $cacheTag;
+    }
+
+    #[On('refresh-widgets')]
+    public function refreshWidget()
+    {
+        $this->dispatch('$refresh');
+    }
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->query(
+                function () {
+                    return User::with('loginsCount');
+                }
+            )
+            ->columns([
+                TextColumn::make('name')
+                    ->searchable(),
+                TextColumn::make('email'),
+                TextColumn::make('sisid')
+                    ->getStateUsing(function ($record) {
+                        return $record->first_login_at ? 'Yes' : 'No';
+                    })
+                    ->badge()
+                    ->color(fn (string $state): string => match ($state) {
+                        'Yes' => 'success',
+                        'No' => 'danger',
+                    })
+                    ->label('Logged In'),
+                TextColumn::make('first_login_at')
+                    ->label('First Login')
+                    ->date('m-d-Y'),
+                TextColumn::make('last_logged_in_at')
+                    ->label('Last Login')
+                    ->date('m-d-Y'),
+                TextColumn::make('loginsCount.count')
+                    ->label('Total Logins')
+                    ->default(0),
+            ])
+            ->filters([
+                SelectFilter::make('User')
```

Comment:
The id of this select filter does not make sense. It should be something like `has_logged_in`